app/lib/theme/widgets/custom/CommunitySourceWidget.tsx

```diff
@@ -0,0 +1,157 @@
+import { WidgetProps } from '@rjsf/utils';
+import { useEffect, useState } from 'react';
+import Autocomplete from '@mui/material/Autocomplete';
+import { TextField } from '@mui/material';
+import styled from 'styled-components';
+
+interface CommunitySourceWidgetProps extends WidgetProps {
+  children: React.ReactNode;
+}
+
+const StyledDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+  margin-top: 8px;
+`;
+
+const CommunitySourceWidget: React.FC<CommunitySourceWidgetProps> = (props) => {
```

Comment:
Another thing noticed, duplicate records can be added, 
![image](https://github.com/user-attachments/assets/0bc56d62-037b-410a-aac5-0f3a98bf396c)

Reply:
Should be fixed now